src/_gettsim/functions_loader.py

```diff
@@ -291,7 +321,7 @@ def _format_duplicated_functions(duplicated_functions, functions, source):
     return "\n".join(lines)
 
 
-def _load_aggregation_combined_dict_from_strings(sources):
+def _load_aggregation_and_link_combined_dict_from_strings(sources):
```

Comment:
```suggestion
def _load_aggregation_and_link_dicts_from_modules(sources):
```

I think this makes clearer what is going on. With the two things, I was thinking of combining aggregations and links...